org.kde.SymbolEditor.json

```diff
@@ -0,0 +1,41 @@
+{
+    "id": "org.kde.SymbolEditor",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15-21.08",
+    "sdk": "org.kde.Sdk",
+    "command": "SymbolEditor",
+    "rename-icon": "symboleditor",
+    "finish-args": [
+        "--device=dri",
+        "--share=ipc",
+        "--socket=cups",
+        "--socket=wayland",
+        "--socket=fallback-x11",
+        "--system-talk-name=org.freedesktop.UDisks2"
+    ],
+    "modules": [
+        {
+            "name": "symboleditor",
+            "buildsystem": "cmake-ninja",
+            "post-install": [
+                "install -Dm0644 logo.png /app/share/icons/hicolor/128x128/apps/org.kde.SymbolEditor.png"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://download.kde.org/stable/symboleditor/2.1.0/symboleditor-2.1.0.tar.xz",
+                    "sha256": "c22f613d3b4873c13d2d7229be9cfadb564fa2aca6696a42b00fcf9d87700f6a"
+                },
+                {
+                    "type": "file",
+                    "url": "https://invent.kde.org/graphics/symboleditor/-/raw/master/logo.png",
```

Comment:
this should be pinned to a specific commit. Otherwise any change on master could break the build.